server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -564,6 +564,16 @@ exports.listLtpsAndFcs = function (user, originator, xCorrelator, traceIndicator
               }
             }
           }
+          else if (layerProtocolName == LayerProtocol.layerProtocolNameEnum.ES_CLIENT) {
+            let elsticSearchClientInterface = layerProtocolInstance[onfAttributes.LAYER_PROTOCOL.ES_CLIENT_INTERFACE_PAC];
+            if ( elsticSearchClientInterface !== undefined) {
+              let elasticSearchValue = elsticSearchClientInterface[onfAttributes.ES_CLIENT.CONFIGURATION]
```

Comment:
Variable name should relate to the value it holds. Please change it from elasticSearchValue to elasticSearchConfiguration or if you have a better idea do so. Thanks.

Reply:
Ok Done